content/en/docs/setup/independent/high-availability.md

```diff
@@ -232,218 +173,158 @@ done
           certSANs:
           - "LOAD_BALANCER_DNS"
         controlPlaneEndpoint: "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT"
-        etcd:
-          local:
-            extraArgs:
-              listen-client-urls: "https://127.0.0.1:2379,https://CP1_IP:2379"
-              advertise-client-urls: "https://CP1_IP:2379"
-              listen-peer-urls: "https://CP1_IP:2380"
-              initial-advertise-peer-urls: "https://CP1_IP:2380"
-              initial-cluster: "CP0_HOSTNAME=https://CP0_IP:2380,CP1_HOSTNAME=https://CP1_IP:2380"
-              initial-cluster-state: existing
-            serverCertSANs:
-              - CP1_HOSTNAME
-              - CP1_IP
-            peerCertSANs:
-              - CP1_HOSTNAME
-              - CP1_IP
-        networking:
+          networking:
             # This CIDR is a calico default. Substitute or remove for your CNI provider.
             podSubnet: "192.168.0.0/16"
 
-1.  Replace the following variables in the template with the appropriate values for your cluster:
+    - `kubernetesVersion` should be set to the Kubernetes version to use. This
+  example uses `stable`.
+    - `controlPlaneEndpoint` should match the address or DNS and port of the load balancer.
+    - It's recommended that the versions of kubeadm, kubelet, kubectl and Kubernetes match.
 
-    - `LOAD_BALANCER_DNS`
-    - `LOAD_BALANCER_PORT`
-    - `CP0_HOSTNAME`
-    - `CP0_IP`
-    - `CP1_HOSTNAME`
-    - `CP1_IP`
-
-1.  Move the copied files to the correct locations:
+1.  Make sure that the node is in a clean state:
 
     ```sh
-    USER=ubuntu # customizable
-    mkdir -p /etc/kubernetes/pki/etcd
-    mv /home/${USER}/ca.crt /etc/kubernetes/pki/
-    mv /home/${USER}/ca.key /etc/kubernetes/pki/
-    mv /home/${USER}/sa.pub /etc/kubernetes/pki/
-    mv /home/${USER}/sa.key /etc/kubernetes/pki/
-    mv /home/${USER}/front-proxy-ca.crt /etc/kubernetes/pki/
-    mv /home/${USER}/front-proxy-ca.key /etc/kubernetes/pki/
-    mv /home/${USER}/etcd-ca.crt /etc/kubernetes/pki/etcd/ca.crt
-    mv /home/${USER}/etcd-ca.key /etc/kubernetes/pki/etcd/ca.key
-    mv /home/${USER}/admin.conf /etc/kubernetes/admin.conf
+    sudo kubeadm reset -f
```

Comment:
Seems odd to tell the user to run `kubeadm reset` for a set of installation instructions. Seems more in line for an FAQ section for how to recover from an issue rather than baked into the default set of instructions.